runtime/worker.rs

```diff
@@ -243,10 +250,18 @@ impl MainWorker {
 
   /// Create new inspector session. This function panics if Worker
   /// was not configured to create inspector.
-  pub fn create_inspector_session(&mut self) -> Box<InspectorSession> {
-    let inspector = self.inspector.as_mut().unwrap();
-
-    InspectorSession::new(&mut **inspector)
+  pub async fn create_inspector_session(&mut self) -> LocalInspectorSession {
+    let inspector = self.inspector.as_ref().unwrap();
+    let mut session = inspector.create_local_session();
+    // Send message to session to complete the handshake.
+    let msg_fut = session
+      .post_message("Runtime.runIfWaitingForDebugger", None)
```

Comment:
I'm surprised this works without specifying the execution context id, REPL had issues using the implicit ID.

Reply:
Context id is hardcoded to "1" throughout inspector implementation. We should look into making them parametrized once `deno_core` gets support for multiple contexts

Reply:
> Context id is hardcoded to "1" throughout inspector implementation.

Yes it is but most messages do still need that id passed to it in the payload.

The inspector has the idea of the implicit "current" context which is used when the optional execution context id is not passed, but we don't implement a way to choose which context is the main context right now.

It's coming back to me a little bit, as far as I can recall global state messages do work without the execution context id but things like evaluate do not.

Resolved by avoidance which is better so much happier with that, we'll get to the missing inspector bits later.

Reply:
I guess we could split the hair in two and add context id from the associated constant to all messages in the follow up PR?

Reply:
> I guess we could split the hair in two and add context id from the associated constant to all messages in the 
Technically its a bug we need to fix, not specifying it *should* work but I went off on a tangent here.

All is good for merge.

Reply:
Okay, thanks, let's address this issue in the future then